client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -59,27 +59,81 @@ export const filterBusinessExtensions = ( extensionInstallationOptions ) => {
 	);
 };
 
+const timeBoxes = [
```

Comment:
Timebox is an interesting name.  Not super opinionated on this, but maybe `timeFrames`?

Reply:
Done here: https://github.com/woocommerce/woocommerce-admin/pull/8042/commits/8ca96a069b8947a94f657c203e3039dc573e9e29